src/BenchmarkDotNet/ConsoleArguments/CommandLineOptions.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using BenchmarkDotNet.Mathematics;
+using CommandLine;
+using CommandLine.Text;
+
+namespace BenchmarkDotNet.ConsoleArguments
+{
+    public class CommandLineOptions
+    {
+        [Option('j', "job", Required = false, Default = "Default", HelpText = "Dry/Short/Medium/Long or Default")]
+        public string BaseJob { get; set; }
```

Comment:
>string [](start = 15, length = 6)

Here I would use an `enum` (`JobType`?), then Default = JobType.Default.
The same for the options below.